### ISSUE TITLE
Change disclaimer to Material-UI dialog 

### ISSUE THREAD

**Issue by AdhamAH** [COVID-19-electronic-health-system/Corona-tracker#211]
## Issue

As we are moving to be using Material-UI, we should restyle the Disclaimer to utilize the Material-UI library

## Motivation 

To make the transition to Material-UI and keep consistency in the app

**Comment by Afrogrrrl** [comment]
On it!

**Comment by SomeMoosery** [comment]
I'm a bit confused - is the purpose of this issue to stylize the Disclaimer with Material-UI? 

If so, I may be reiterating myself a bit now but want to keep things clean - when fixing these we should still make two separate PRs - one for saving disclaimer state (that fixes #205), and another for restyling (solves this one)

**Comment by Afrogrrrl** [comment]
Yes I'll be making a separate PR for #205 and one for this one. Sorry that wasn't clear

**Comment by AdhamAH** [comment]
It is my way of explaining things @SomeMoosery I should say it is separate. sorry

**Comment by SomeMoosery** [comment]
Perfect - sounds good! I figured that was the case, but just wanted to be sure on my end!

I also quickly edited this Issue just because, moving forward into the weekend when we release a unified contribution process, we'll be only giving contextual information and acceptance criteria in issues, removing anything relating to conversations/background/etc to keep things as clean as possible. Just thought I'd explain why I edited! :)
